recipes/pyo-oracle/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "pyo-oracle" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bio-oracle/pyo_oracle/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: b087ad5eb5e24c456a8cc0322ef084769d9315159b5f7b9da02b468a3fee8b16
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools >=61.0
+    - pip
+  run:
+    - python >=3.7
+    - erddapy
+
+test:
+  imports:
+    - pyo_oracle
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/vinisalazar/metaphor
```

Should it be the following?
```suggestion
  home: https://github.com/bio-oracle/pyo_oracle
```